Convert parameters to hash

RustComet opened the pull request:
This allows us to continue to use `.except!`

RustComet committed:
Convert parameters to hash

This allows us to continue to use `.except!`

knightstick commented on app/controllers/shortener/shortened_urls_controller.rb:
```diff
@@ -18,7 +18,7 @@ def show
       end
 
-      params.except! *[:id, :action, :controller]
+      params.to_h.except! *[:id, :action, :controller]
```
I'm not sure this is going to work - doesn't this make a new hash that's a "copy" of params and mutate it?

RustComet replied:
Good catch. You're right

RustComet committed:
Assign filtered_params to variable and pass that around

As Chris pointed out, just calling `._h` on the params just modifies the hash representation, not the params themselves.

So if we set the modified params to a variable, we can pass that around instead.
